--- jwhois-4.0/configure.ac.orig	2003-06-30 19:12:14.000000000 +0200
+++ jwhois-4.0/configure.ac	2003-07-26 12:55:24.000000000 +0200
@@ -81,11 +81,11 @@
 
 AC_C_BIGENDIAN
 
-AC_CHECK_FUNC(connect,,
-  AC_CHECK_LIB(socket, connect))
-AC_CHECK_FUNC(socket,,
+AC_CHECK_FUNC(connect,,[
+  AC_CHECK_LIB(socket, connect)])
+AC_CHECK_FUNC(socket,,[
   AC_CHECK_LIB(socket, socket)
-  AC_CHECK_LIB(inet, socket))
+  AC_CHECK_LIB(inet, socket)])
 AC_CHECK_FUNCS(getipnodebyname)
 AC_CHECK_FUNC(getaddrinfo,
   [AC_DEFINE(HAVE_GETADDRINFO,1)
@@ -94,9 +94,6 @@
      AC_CHECK_LIB(nsl, gethostbyname))
    AC_CHECK_FUNC(inet_aton)])
 
-AC_CHECK_LIB(inet6, main,
-  AC_CHECK_LIB(resolv, main))
-
 AC_CHECK_FUNCS(memcpy strtol)
 AC_CHECK_FUNCS(strcasecmp strncasecmp getopt_long)
 AC_STDC_HEADERS
